src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Script.java

```diff
@@ -102,11 +102,11 @@ public Script appendStringLiteral(CharSequence text, char quoteChar) {
      * Returns a "live" view of the script as a {@code Content} object.
```

Comment:
Don't change it now, but the comment abut a "live view" is sort-of unnecessary because in general most HTMLTree objects are mutable.